app/Http/Controllers/Api/V1/CurrikiGo/LmsController.php

```diff
@@ -66,4 +69,37 @@ public function projects(Request $request)
             'projects' => ProjectPublicResource::collection($projects),
         ], 200);
     }
+
+    public function activities(Request $request)
+    {
```

Comment:
It would be better to use Request validation here.
I see that it's a TODO item for the other handlers in this controller as well.